nomad/structs/structs.go

```diff
@@ -2614,6 +2620,9 @@ func NewRestartPolicy(jobType string) *RestartPolicy {
 }
 
 const ReschedulePolicyMinInterval = 15 * time.Second
+const ReschedulePolicyMinDelay = 5 * time.Second
+
+var RescheduleDelayFunctions = [...]string{"linear", "exponential", "fibonacci"}
```

Comment:
Why the `...`?

Reply:
It's just syntactic sugar to let the compiler figure out the length of the array.  `[3]string{"linear", "exponential", "fibonacci"}` is correct too, but with the variadic syntax if we add more in the future I don't have to change the number.